week-2-practise/level-1/5-indexOf.js

```diff
@@ -3,12 +3,12 @@
 
 var itemToFind = 7;
 var arr = [3, 5, 61, 7, 123];
-var index; // ONLY EDIT THIS LINE
+var index = itemToFind.indexOf (7); // ONLY EDIT THIS LINE
 
 console.log(index);
 
 /* 
   EXPECTED RESULT
   ---------------
   3
-*/
+*
```

Comment:
Does this work without any errors if you run it? You've accidentally deleted the `/` from the comment end marker.